keymaps/intellij-idea-keymap.cson

```diff
@@ -113,13 +140,54 @@
   # Recent files popup
   'cmd-e': 'fuzzy-finder:toggle-buffer-finder'
 
+  # Reveal active file
+  'cmd-ctrl-s': 'tree-view:reveal-active-file'
+
+  # Show bookmarks
+  'cmd-f3': 'bookmarks:view-all'
+
+  # Toggle bookmark
+  'f3': 'bookmarks:toggle-bookmark'
+
   ##################
   # Live Templates #
   ##################
 
   # Insert live templates
   'cmd-j': 'snippets:available'
 
+'.platform-darwin .tree-view':
+
+  ###########
+  # General #
+  ###########
+
+  'left': 'tree-view:collapse-directory'
```

Comment:
This shortcut already works out of the box in Atom. I would remove it here such that we just have the shortcuts which are different between IDEA and Atom.

This is also the case for `right` and `enter`.